test/extension.test.ts

```diff
@@ -762,3 +780,41 @@ suite('Compilation info', () => {
     expect(info.compiler).to.eq('cl.exe');
   });
 });
+
+suite('CMake System tests', () => {
+  (process.env.HasVs == 'true' ? suite.skip : suite)('No present kit',() => {
+        let path_backup = "";
```

Comment:
I think we should use `''` for all strings.

Reply:
Done